Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.yml

```diff
@@ -2385,18 +2384,16 @@ script:
     - contextPath: SentinelOne.Notes.UpdatedAt
       description: The note updated time.
       type: string
-  dockerimage: demisto/python3:3.10.13.83255
+  dockerimage: demisto/python3:3.10.13.84405
   isfetch: true
   ismappable: true
   isremotesyncin: true
   isremotesyncout: true
   script: ""
   type: python
   subtype: python3
+  runonce: false
 tests:
 - SentinelOne V2.1 - Test
 - SentinelOne V2.0 - Test
 fromversion: 5.0.0
-defaultclassifier: SentinelOne Classifier
```

Comment:
lines 2400-2402: I did not intentionally remove these

Reply:
Hi @randomizerxd, can you please revert this change. we need those default values